### PR TITLE
Upgrade to .Net Core 3.1

### PR DESCRIPTION
**Pull request by sutser**

### CODE REVIEW

**Review by tautropfli — CHANGES_REQUESTED** [review]
While you're at it, could you install paket using the tools manifest?

**Comment by sutser** [comment]
> 
> 
> While you're at it, could you install paket using the tools manifest?

Addressed in: https://github.com/messerli-informatik-ag/query-provider/pull/8/commits/aae53ba89d8d61a1c000c7cf714a297c22b6be82

File: .paket/Paket.Restore.targets [hunk 1]
```diff
@@ -20,18 +20,18 @@
     <PaketBootstrapperStyle Condition="Exists('$(PaketToolsPath)paket.bootstrapper.proj')">proj</PaketBootstrapperStyle>
     <PaketExeImage>assembly</PaketExeImage>
     <PaketExeImage Condition=" '$(PaketBootstrapperStyle)' == 'proj' ">native</PaketExeImage>
-    <MonoPath Condition="'$(MonoPath)' == '' And Exists('/Library/Frameworks/Mono.framework/Commands/mono')">/Library/Frameworks/Mono.framework/Commands/mono</MonoPath>
+    <MonoPath Condition="'$(MonoPath)' == '' AND Exists('/Library/Frameworks/Mono.framework/Commands/mono')">/Library/Frameworks/Mono.framework/Commands/mono</MonoPath>
```
Comment (tautropfli) [inline .paket/Paket.Restore.targets:23]: Run `dotnet paket install`, I think this file was generated with a different version of paket.

**Review by tautropfli — CHANGES_REQUESTED** [review]

File: .paket/Paket.Restore.targets (hunk 1 above)
Comment (sutser) [reply .paket/Paket.Restore.targets:23]: No, it was generated with dotnet paket install

**Review by tautropfli — APPROVED** [review]

**Pull request merged**